stan/math/prim/scal/prob/neg_binomial_lccdf.hpp

```diff
@@ -37,7 +37,7 @@ return_type_t<T_shape, T_inv_scale> neg_binomial_lccdf(
   scalar_seq_view<T_n> n_vec(n);
   scalar_seq_view<T_shape> alpha_vec(alpha);
   scalar_seq_view<T_inv_scale> beta_vec(beta);
-  size_t size = max_size(n, alpha, beta);
+  size_t max_size_seq_view = max_size(n, alpha, beta);
```

Comment:
By the way, `max_size_seq_view` is really so much clearer than just `size`, and the consistency across files is super nice. :)

Reply:
Agreed. The size was really ambigious.